go2_robot_sdk/config/nav2_params_docs/global_costmap.md

```
@@ -0,0 +1,35 @@
+# Global Costmap
+
+**Global Costmap 튜닝 전(원본) 값 요약**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)